ape_aws/kms/_cli.py

```diff
@@ -54,7 +63,103 @@ def create_key(
 
 
-# TODO: Add `ape aws kms import`
+@kms.command(name="import")
+@ape_cli_context()
+@click.option(
+    "-p",
+    "--private-key",
+    "private_key_path",
+    type=click.Path(),
+    help="The private key you intend to import",
+    metavar="Path",
+)
+@click.option(
+    "-a",
+    "--admin",
+    "administrators",
+    multiple=True,
+    help="Apply key policy to a list of administrators if applicable, ex. -a ARN1, -a ARN2",
+    metavar="list[ARN]",
+)
+@click.option(
+    "-u",
+    "--user",
+    "users",
+    multiple=True,
+    help="Apply key policy to a list of users if applicable, ex. -u ARN1, -u ARN2",
+    metavar="list[ARN]",
+)
+@click.option(
+    "-d",
+    "--description",
+    "description",
+    help="The description of the key you intend to create.",
+    metavar="str",
+)
+@click.option(
+    "--use-mnemonic",
+    "import_from_mnemonic",
+    help="Import a key from a mnemonic phrase",
+    is_flag=True,
+)
+@click.option(
+    "--hd-path",
+    "hd_path",
+    help="The hierarchical deterministic path to derive the key from",
+    metavar="str",
+)
+@click.argument("alias_name")
+def import_key(
+    cli_ctx,
+    alias_name: str,
+    private_key_path: Path,
+    administrators: list[str],
+    users: list[str],
+    description: str,
+    import_from_mnemonic: bool,
+    hd_path: str,
+):
+    if private_key_path:
+        if isinstance(private_key_path, str):
+            private_key_path = Path(private_key_path)
+        if private_key_path.exists() and private_key_path.is_file():
+            cli_ctx.logger.info(f"Reading private key from {private_key_path}")
+            private_key = private_key_path.read_text().strip()
+
+    elif import_from_mnemonic:
+        if not hd_path:
+            hd_path = ETHEREUM_DEFAULT_PATH
+        mnemonic = click.prompt("Enter your mnemonic phrase", hide_input=True)
+        EthAccount.enable_unaudited_hdwallet_features()
+        account = EthAccount.from_mnemonic(mnemonic, account_path=hd_path)
+        private_key = account.key.hex()
+
+    else:
+        private_key = click.prompt("Enter your private key", hide_input=True)
+
+    key_spec = ImportKeyRequest(
+        alias=alias_name,
+        description=description,  # type: ignore
+        admins=administrators,
+        users=users,
+    )
+    key_id = kms_client.create_key(key_spec)
+    create_key_response = kms_client.get_parameters(key_id)
+    public_key = create_key_response["PublicKey"]
+    import_token = create_key_response["ImportToken"]
+    import_key_spec = ImportKey(
+        **key_spec.model_dump(),
+        key_id=key_id,  # type: ignore
+        public_key=public_key,  # type: ignore
+        private_key=private_key,  # type: ignore
+        import_token=import_token,  # type: ignore
+    )
+    response = kms_client.import_key(import_key_spec)
+    if response["ResponseMetadata"]["HTTPStatusCode"] != 200:
+        cli_ctx.abort("Key failed to import into KMS")
```

Comment:
Does it give any message in case of a failure? might want to display it

Reply:
Well yeah, that's what the above line will give you

I'm asking if there's any field in `response` that will help you understand the issue that came up which we can print w/ `.abort`

Reply:
ohhhhh, let me check what we can give to the user

Reply:
boto3 seems to catch most of these errors, I tried to get a bad response, but I believe boto3 is handling the errors. To be sure, this is how I'm handling it